api/models.py

```diff
@@ -5,22 +5,21 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			('SLEEPYCAT', 'Sleepy Cat'),
+			('LAZYCAT', 'Lazy Cat'),
+			('ACTIVECAT', 'Active Cat'),
+			('MENTORCAT', 'Mentor Cat'),
 	)
 
 	user = models.OneToOneField(User) 
-	#id = models.AutoField(primary_key=True) 
 	fullname = models.CharField(max_length=100, blank=False)
 	githubuser = models.CharField(max_length=100, null=False, blank=False)
 	email = models.EmailField(blank=False)
 	profile_picture = models.ImageField(blank = True)  
-	website = models.URLField(blank = True)
+	website = models.URLField(blank = False)
```

Comment:
You are using spaces int the beginning and end of `=` in this line and some others, but others you are not.